landoui/treestatus.py

```diff
@@ -227,13 +227,13 @@ def treestatus_tree(tree: str):
             raise
 
         error = f"Error received from LandoAPI: {exc.detail}"
-        logger.error(error)
+        logger.info(error)
```

Comment:
I think for this one, given that we've identified that there are lots of 404s due to spam, we should probably just return a 404 immediately (and not redirect or flash a message) in that case. Otherwise we may still be flooding the logs with useless data.